web-console/src/lib/components/streaming/management/PipelineTable.tsx

```diff
@@ -507,6 +509,16 @@ export default function PipelineTable() {
     setExpandedRows(newExpandedRows)
   }
 
+  const [columnVisibilityModel, setColumnVisibilityModel] = useLocalStorage<GridColumnVisibilityModel>({
```

Comment:
seems thisu pattern is duplicated in InspectionTable and TableSqlPrograms, worth factoring it in a common method?